### PR TITLE
Fix #5251 Modify Feature Flag Names to reflect those on Oppia Web

### LINKED ISSUE #5251: Feature Request: Modify Feature Flag Names to reflect those on Oppia Web
### Is your feature request related to a problem? Please describe.

Some of the feature flags used to gate feature on Oppia-Android need to be updated to  reflect new android-specific feature flag names from oppia-web so that remote enabling and disabling of these features is possible.

### Describe the solution you'd like

The feature flag names in the [FeatureFlagConstants](https://github.com/oppia/oppia-android/blob/develop/utility/src/main/java/org/oppia/android/util/platformparameter/FeatureFlagConstants.kt) file need to be modified and to the new respective names as shown below.

| Feature Flag | Current Feature Flag Name | Proposed Feature Flag Name |
| ------------- | ------------------ | ------------------ |
| EnableDownloadsSupport | downloads_support | android_enable_downloads_support |
| EnableExtraTopicTabsUi | extra_topic_tabs_ui | android_enable_extra_topic_tabs_ui |
| EnableLearnerStudyAnalytics | learner_study_analytics | android_enable_learner_study_analytics |
| EnableFastLanguageSwitchingInLesson | fast_language_switching_in_lesson | android_enable_fast_language_switching_in_lesson |
| EnableLoggingLearnerStudyIds | logging_learner_study_ids | android_enable_logging_learner_study_ids |
| EnableEditAccountsOptionsUi | edit_accounts_options_ui | android_enable_edit_accounts_options_ui |
| EnablePerformanceMetricsCollection | performance_metrics_collection |  android_enable_performance_metrics_collection |
| EnableSpotlightUi | spotlight_ui | android_enable_spotlight_ui |
| EnableInteractionConfigChangeStateRetention | interaction_config_change_state_retention | android_enable_interaction_config_change_state_retention |
| EnableAppAndOsDeprecation | app_and_os_deprecation | android_enable_app_and_os_deprecation |

### Describe alternatives you've considered

N/A

### Additional context

N/A

### PR DESCRIPTION
**Pull request by Tejas-67**
## Explanation
Modified feature flag names to respective new names as described [here](https://github.com/oppia/oppia-android/issues/5251).

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- For PRs introducing new UI elements or color changes, both light and dark mode screenshots must be included
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt [hunk 1]
```diff
@@ -46,12 +46,10 @@ class PoliciesFragmentPresenter @Inject constructor(
...
 
     if (policyPage == PolicyPage.PRIVACY_POLICY) {
-      policyDescription =
-        resourceHandler.getStringInLocale(R.string.privacy_policy_content)
+      policyDescription = resourceHandler.getStringInLocale(R.string.privacy_policy_content)
       policyWebLink = resourceHandler.getStringInLocale(R.string.privacy_policy_web_link)
     } else if (policyPage == PolicyPage.TERMS_OF_SERVICE) {
-      policyDescription =
-        resourceHandler.getStringInLocale(R.string.terms_of_service_content)
+      policyDescription = resourceHandler.getStringInLocale(R.string.terms_of_service_content)
```
Comment (adhiamboperes) [inline app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt:52]: These changes are not related to this PR. Please revert

File: app/src/main/res/values/terms_of_service.xml [hunk 2]
```diff
@@ -236,4 +236,4 @@
       <li>26 Sep 2015: Revised terms added.</li>
     </ul>
 </]]></string>
```
Comment (adhiamboperes) [inline app/src/main/res/values/terms_of_service.xml:238]: This file change too is not related to this PR.

File: utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt
Comment (adhiamboperes) [inline utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt:1]: This file should not be part of this PR

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @Tejas-67!

Your changes on the FeatureFlagConstants look good, but the remaining file changes need to be reverted.

Whenever creating a new PR, you should create a new branch based off of the develop branch. You can do this by:
1. `git checkout develop` checkout your local develop
2. `git pull upstream develop` sync with upstream
3. `git checkout -b new-branch-name` create clean branch

In case you had mistakenly made commits on you local develop branch, you can do a hard reset or delete the branch and checkout a clean one from upstream.

**Comment by Tejas-67** [comment]
@adhiamboperes Okay, I will close this PR and create a fresh one.
